### DISCUSSION TITLE
Check inverter more frequently

### DISCUSSION THREAD

**Discussion by lizdodd** [SBFspot/SBFspot#516] [Q&A]
I'd like to check the inverter more frequently, every minute, because I need to cut back consumption when clouds pass over the sun. I've tried to run the SBFspot cronjob every minute as well as the personal script which feeds MQTT every minute but I still get 5 consecutive readings which are identical, changing after the fifth minute
Like this
>  `2316.0
2316.0
2316.0
2316.0
2316.0
1956.0
1956.0
1956.0
1956.0
1956.0`

What else can I do to get a reading every minute?

**Comment by sbf-** [comment]
Spotdata and inverters tables are updated at every run. I guess you’re querying daydata table which has only 5min records.
